tests/benchmark/test_runner.py

```diff
@@ -41,9 +41,10 @@ def test_get_coroutines(benchmark: BenchmarkFixture, catalog: AntaCatalog, inven
     selected_tests = prepare_tests(inventory=inventory, catalog=catalog, tests=None, tags=None)
 
     assert selected_tests is not None
+    results = ResultManager()
```

Comment:
is this not making the ResultManager unique across all runs?

Reply:
Good catch